content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/clientlibs/editor/js/embed.js

```diff
@@ -238,17 +237,14 @@
      * Toggles the disabled state and visibility of tabs linked to panels matching the target.
      * Tabs that match the provided value are enabled / shown, otherwise they are disabled / hidden.
      *
-     * @param {String} panelSelectors Comma separated list of panel selectors for which the tabs should be toggled
-     * @param {String} value The value of the target to enable and show
+     * @param {String[]} values The values of the target to enable and show
      */
-    function toggleShowHideTabs(panelSelectors, value) {
-        var panelElements = document.querySelectorAll(panelSelectors);
-
-        for (var i = 0; i < panelElements.length; i++) {
-            var panelElement = panelElements[i];
-            var showHideTargetValue = getShowHideTargetValue(panelElement);
-            var tabElement = getTabElementForPanel(panelElement);
-            if (showHideTargetValue === value) {
+    function toggleShowHideTabs(values) {
+        var panelElement = document.querySelectorAll("[data-cmp-embed-dialog-edit-embeddableoptions]");
```

Comment:
I'd name it `panelElements` (plural) as it can contain several elements.

Reply:
changed in 1d10a84c2ff4c13074767a5a7aea58f71deac6cf